hw/dv/tools/dvsim/verilator.hjson

```diff
@@ -108,6 +108,12 @@
   cov_db_test_dir: ""
 
   build_modes: [
+    {
+      name: verilator_gui
```

Comment:
This seems a bit silly. Could we do it the other way, where we check whether `$tool_gui` exists as a build mode, and only use it if so?

Reply:
If a mode is enabled but it doesn't exist, dvsim throws an error and exits (which is the correct behavior). How to deal with what you are asking for probably does not belong with this PR. Added a TODO to figure it out later.

Reply:
Understood. Presumably, the error currently looks like some weird internal "aargh, something went wrong" message. It would be nice (as a future change) to print out something like "This tool doesn't have a GUI" to make it clear to the user what's going on.